### PR TITLE
Removed sageconv dropout due to not existing anymore(?)

### PR DESCRIPTION
**Pull request by MarcusLoppe**
Seems like the torch_geometric dropped or change the args for SAGEConv so sageconv_dropout is no longer a valid arg.

@lucidrains  Please approve this pull request since you are not able to create the autoencoder without removing this arg.
I've seen no improvements in accuracy when using the dropout so it probably isn't needed or very useful.
Thank you 😄

### CODE REVIEW

**Comment by lucidrains** [comment]
oh thanks Marcus!

File: meshgpt_pytorch/meshgpt_pytorch.py [hunk 1]
```diff
@@ -484,7 +483,7 @@ def __init__(
 
         # initial sage conv
 
-        sageconv_kwargs = {**sageconv_kwargs, 'sageconv_dropout' : sageconv_dropout}
+        sageconv_kwargs = {**sageconv_kwargs}
```
Comment (lucidrains) [inline meshgpt_pytorch/meshgpt_pytorch.py:486]: you can just remove this line altogether

Comment (MarcusLoppe) [reply meshgpt_pytorch/meshgpt_pytorch.py]: > you can just remove this line altogether

All done :) I also bumped the version

File: meshgpt_pytorch/meshgpt_pytorch.py [hunk 2]
```diff
@@ -482,14 +481,11 @@ def __init__(
...
-        # initial sage conv
-
-        sageconv_kwargs = {**sageconv_kwargs, 'sageconv_dropout' : sageconv_dropout}
-
+        # initial sage conv 
         init_encoder_dim, *encoder_dims_through_depth = encoder_dims_through_depth
         curr_dim = init_encoder_dim
 
-        self.init_sage_conv = SAGEConv(dim_codebook, init_encoder_dim, **sageconv_kwargs)
+        self.init_sage_conv = SAGEConv(dim_codebook, init_encoder_dim, {**sageconv_kwargs})
```
Comment (lucidrains) [inline meshgpt_pytorch/meshgpt_pytorch.py:488]: @MarcusLoppe oh I think this line can be kept the same, unless something changed with torch geometric

Comment (MarcusLoppe) [reply meshgpt_pytorch/meshgpt_pytorch.py]: Probably true, just didn't want to mess something up if it expected like a dict of a dict :)
Or maybe I'm just bad at python and that actually made the dicts concat/merges.
But I amended as per your comment

**Pull request merged**